internal/utils/telegraf/measurement/measurement.go

```diff
@@ -0,0 +1,18 @@
+package measurement
+
+import (
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+)
+
+// Measurement is a storage struct for storing measurements from telegraf
+// plugins
+type Measurement struct {
+	Measurement        string
+	Fields             map[string]interface{}
+	Tags               map[string]string
+	MetricType         datapoint.MetricType
+	OriginalMetricType string
+	T                  []time.Time
```

Comment:
`T` is ambiguous.  I might guess it stood for "timestamp" but I wouldn't guess it meant a slice of times.